src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -52,26 +52,15 @@
     private static final FeedbackResponsesDb frDb = new FeedbackResponsesDb();
     private static final FeedbackResponseCommentsDb fcDb = new FeedbackResponseCommentsDb();
     
-    public String putDocumentsForStudents(DataBundle dataBundle) {
-        for (StudentAttributes student : dataBundle.students.values()) {
-            student = getStudentForEmail(student.course, student.email);
-            putDocument(student);
-            ThreadHelper.waitFor(50);
-        }
-        
-        return Const.StatusCodes.BACKDOOR_STATUS_SUCCESS;
-    }
-    
     /**
      * Persists given data in the datastore Works ONLY if the data is correct.
      *  //Any existing copies of the data in the datastore will be overwritten.
-     *      - edit: use removeDataBundle/deleteExistingData to remove.
+     *      - edit: use removeDataBundle to remove.
      *              made this change for speed when deletion is not necessary.
```

Comment:
I will be creating another issue to tidy up the documentation of this BackDoor slice, and this change can be part of that.